tracer/src/Datadog.Trace/Propagators/DatadogContextPropagator.cs

```diff
@@ -46,7 +46,7 @@ samplingPriority.Value switch
             var propagationHeaderMaxLength = context.TraceContext?.Tracer.Settings.TagPropagationHeaderMaxLength ?? TagPropagation.OutgoingPropagationHeaderMaxLength;
             var propagatedTraceTags = context.TraceContext?.Tags.ToPropagationHeader(propagationHeaderMaxLength) ?? context.PropagatedTags;
 
-            if (propagatedTraceTags != null)
+            if (!string.IsNullOrEmpty(propagatedTraceTags))
```

Comment:
Just so I understand, this is necessary because _if_ `propagationHeaderMaxLength` is 0, then `context.TraceContext?.Tags.ToPropagationHeader(propagationHeaderMaxLength)` returns `string.Empty`, right?

Reply:
Yes actually [ToPropagationHeader](https://github.com/DataDog/dd-trace-dotnet/blob/75deafac76cff7e19806d7fd9a5947c5d41b094d/tracer/src/Datadog.Trace/Tagging/TagPropagation.cs#L127) will return an empty string in several cases:
- when we set header length to 0
- when header is bigger than maximum headerlength
- or if there were no tags to add actually, the returned string will be empty